content/events/cshl-genome-informatics-2023.md

```diff
@@ -0,0 +1,40 @@
+---
+author: "AnVIL"
+conference: "Genome Informatics 2023"
+description: "The AnVIL team will attend the CSHL Genome Informatics meeting to share about the AnVIL platform with the genomics community. This meeting will draw together researchers focused on genomic data science research, for which AnVIL can be a useful platform to leverage and can offer solutions to some common challenges in the field."
+eventType: "Conference"
+featured: true
+hashtag: "#gi2023"
+location: "Cold Spring Harbor, NY, USA"
+sessions: [{sessionStart: "6 December 2023"}, {sessionStart: "7 December 2023"}, {sessionStart: "8 December 2023"}, {sessionStart: "9 December 2023"}]
+timezone: "America/New_York"
+title: "CSHL Genome Informatics 2023"
+---
+
+<event-hero></event-hero>
+
+### Description
+
+The AnVIL team will attend the CSHL Genome Informatics meeting to share about the AnVIL platform with the genomics community. This meeting will draw together researchers focused on genomic data science research, for which AnVIL can be a useful platform to leverage and can offer solutions to some common challenges in the field.
+
+#### Background
+
+The next Cold Spring Harbor Laboratory Conference on Genome Informatics will be held at Cold Spring Harbor, New York (this meeting has alternated with the Hinxton meeting started under the joint auspices of CSHL and the Wellcome Trust). The meeting will begin at 7:30 p.m. on Wednesday, December 6, and finish with lunch on Saturday, December 9, 2023.
+Tentative Meeting Topics:
```

Comment:
@esthermcdade Insert a new line between lines 22 and 23 and view the change on localhost to see the difference.